### PR TITLE
added grid garden for first-timers

### PR DESCRIPTION
**Pull request by jjgaray**
## Contributing

If you want to add a project to this list, please make sure that:

- [ ] You read the [contributing guide](https://github.com/OtacilioN/awesome-hacktoberfest-2020/blob/master/CONTRIBUTING.md)
- [ ] The project can't be just an "add your name" or simply + 1 contribution to hacktoberfest
- [ ] Place them in alphabetical order
- [ ] One repository per PR/contribution - _avoid conflicts and simplify review_
- [ ] Your pull request is not spammy just adding an empty line, otherwise it will be marked as invalid and not count to hacktoberfest.

If they meet the requirements above, feel free to create a PR. Happy Hacking! :sparkles:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@
 - [CompetitiveCode](https://github.com/Vishruth-S/CompetitiveCode)
 - [DevJoke](https://github.com/shrutikapoor08/devjoke)
 - [Developer Experience Knowledge Base](https://github.com/DXHeroes/knowledge-base-content)
+- [Grid-Garden] (https://github.com/thomaspark/gridgarden/)
```
Comment (OtacilioN) [inline README.md:29]: Can you remove the blank space between the name of repository and the link?

**Review by OtacilioN — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jjgaray) [reply README.md:29]: sure

Comment (jjgaray) [reply README.md]: done! :D

**Comment by jjgaray** [comment]
Hi @OtacilioN done fixing

**Pull request closed without merging**